plaidml/edsl/tests/edsl_test-skip.intel_gen.txt

```diff
@@ -5,17 +5,13 @@ CppEdsl.BitNot
 CppEdsl.BitRightScalar
 CppEdsl.ComplexConv2d
 CppEdsl.ConvI8
-CppEdsl.CosH
 CppEdsl.Erf
-CppEdsl.Floor
 CppEdsl.HigherPrecisionInvalidNegative
 CppEdsl.HigherPrecisionConstants
 CppEdsl.MnistCnn
 CppEdsl.Prng
 CppEdsl.Pow
 CppEdsl.Round
```

Comment:
Is CppEdsl.Round still fail after this patch?

Reply:
Um no. I just missed this newly added file.